### PR TITLE
Prevent old_file_browser api calls and grid generation in current fil…

### PR DESCRIPTION
**Pull request by jkmarx**
Resolves issue #2043

### CODE REVIEW

File: refinery/static/source/js/refinery/solr/solr_document_table.js [hunk 1]
```diff
@@ -107,12 +107,10 @@ SolrDocumentTable.prototype.render = function(solrResponse) {
     .trigger('refinery/solrTable/destroy')
     .html("");
    // Required while data set 2 lives in parallel.
-  if (window.location.href.indexOf('data_sets') === -1) {
-    self._renderTable(solrResponse);
+  self._renderTable(solrResponse);
     //$( "#" + self._parentElementId ).html( code );
     // attach event listeners
     // ..
-  }
```
Comment (mccalluc) [inline refinery/static/source/js/refinery/solr/solr_document_table.js]: [Originally](https://github.com/refinery-platform/refinery-platform/blame/46619c1a75bf765ca874e1797e3ea35db3aa0e24/refinery/static/source/js/refinery/solr/solr_document_table.js#L109), this had been:
```
  if (window.location.href.indexOf('data_sets2') === -1) {
    self._renderTable(solrResponse);
    //$( "#" + self._parentElementId ).html( code );
    // attach event listeners
    // ..
}
```

and the intention seemed to be *not* to run it in FB2, but I'm really not sure what it should be at this point.

Comment (jkmarx) [reply refinery/static/source/js/refinery/solr/solr_document_table.js]: @mccalluc The old table generation should only be called when in the data_set old which is provenance. It's hacky, but will work while we are in the transition phase. Open to other suggestions, but note this is temporary.

Comment (jkmarx) [reply refinery/static/source/js/refinery/solr/solr_document_table.js]: I relocated the conditional in the document.ready, so it's no longer needed here.

Comment (mccalluc) [reply refinery/static/source/js/refinery/solr/solr_document_table.js]: ok, I see the connection now. Thanks!

**Pull request merged**